azkaban-common/src/main/java/azkaban/utils/Props.java

```diff
@@ -811,6 +812,30 @@ public void storeFlattened(OutputStream out) throws IOException {
   }
 
   /**
+   * Returns a map of all the flattened properties
+   *
+   * @Return
+   */
+  public Map<String,String> getFlattened(){
```

Comment:
Actually we don't need this method. It gives same function as props.getMapByPrefix("") i.e. prefix with empty string.